modules/Microsoft.Web/sites/readme.md

```diff
@@ -390,6 +390,7 @@ userAssignedIdentities: {
 | `resourceGroupName` | string | The resource group the site was deployed into. |
 | `resourceId` | string | The resource ID of the site. |
 | `systemAssignedPrincipalId` | string | The principal ID of the system assigned identity. |
+| `defaultHostname` | string | Default hostname of the app. |
```

Comment:
Please run the 'Set-ModuleReadMe.ps1' script to regenerate the documentation after the changes.
![image](https://user-images.githubusercontent.com/28486158/185042381-1b62a82b-0a97-42b1-8910-2aedfb6ff1c3.png)

The pipeline static code check fails as it detects a difference between the documentation submitted vs. the document that gets autogenerated during the test for comparison:

This is the error when I attempt to replay your branch changes
![image](https://user-images.githubusercontent.com/28486158/185042929-769b5673-17c2-4064-90ec-24a9bb85e8c3.png)
Here is a guide on how to use the [Set-ModuleReadMe.ps1](https://github.com/Azure/ResourceModules/wiki/Contribution%20guide%20-%20Generate%20module%20Readme) script

Reply:
OK thanks Ahmad, will do 👍

Reply:
@DanielLarsenNZ have you tested the deployment with the new changes?

Reply:
Yes. WOMM. I have an Azure Quickstart template in progress that uses this (patched) module.